py/server/tests/test_table_factory.py

```diff
@@ -64,6 +64,11 @@ def test_time_table(self):
         self.assertTrue(t.is_refreshing)
         self.assertEqual("2021-11-06T13:21:00.000000000 ET", format_datetime(t.j_table.getColumnSource("Timestamp").get(0), time_zone('ET')))
 
+    def test_time_table_blink(self):
+        t = time_table("PT1s", blink_table=True)
+        self.assertEqual(1, len(t.columns))
+        self.assertTrue(_JBlinkTableTools.isBlink(t.j_table))
+
```

Comment:
Would it be better to add a property 'is_blinking' to Table?

Reply:
I think we'd want to add `io.deephaven.engine.table.Table#isBlink` before we added it to the python side. Note: `io.deephaven.engine.table.impl.BaseTable#isBlink` exists, but is scoped to the implementations, not the interface. Should we elevate to engine Table interface @rcaudy?

Reply:
Regardless, I think that would be a new issue / PR.

Reply:
Let's defer adding anything to the table interface. Users who really want to know can look at attributes for now.